### DISCUSSION TITLE
Autocomplete and importing ComposableArchitecture

### DISCUSSION THREAD

**Discussion by fabstu** [pointfreeco/swift-composable-architecture#455] [General]
I observed that for files which have `import ComposableArchitecture` at the first position, autcomplete does not work at all. Moving `import ComposableArchitecture` to the last position does sometimes help for a short moment, then autocomplete breaks again. Do you have the same issue? How did you solve it?

Xcode 12.5 beta3

EDIT1: Now it works again. Might be the usual Xcode autocomplete issues. 🖌️

**Comment by gkaimakas** [comment]
Most probably Xcode's fault. I import TCA first most of the time, haven't had any issues (apart from the normal "Xcode can't autocomplete anything" that happens occasionally)
